charts/rancher-vsphere-cpi/values.yaml

```diff
@@ -81,7 +85,7 @@ versionOverrides:
 
 cloudControllerManager:
   repository: rancher/mirrored-cloud-provider-vsphere-cpi-release-manager
-  tag: v1.22.7
+  tag: latest
```

Comment:
Set the default tag to `latest` to make it obvious when the version overrides are not working properly